phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -33,4 +33,11 @@ cloudProvider: "$CLOUD_PROVIDER"
 EOF
 fi
 
+if [[ "${KUBEPROXY_MODE}" == "ipvs" ]]; then
```

Comment:
We need this variable `KUBEPROXY_MODE` set from other phase1 providers as well.
Can you please add this variable in openstack provider as well.

Reply:
Sure, I added in openstack,

Reply:
we cannot use the variable with curly braces like "${KUBEPROXY_MODE}", as it will conflict with terraform and/or jsonnet templating. See the other variables in the file and follow the similar pattern